resources/assets/components/pages/CampaignsPage/CampaignFilters/FilterInput.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Checkbox input component.
+ *
+ * @param {Object}
+ */
+const FilterInput = ({ filterName, filterValue, handleSelect, isChecked }) => (
+  <label className="flex items-start justify-start pb-2" htmlFor={filterValue}>
+    <input
+      id={filterValue}
+      checked={isChecked}
+      className="mt-1"
+      name={filterValue}
+      onChange={handleSelect}
+      type="checkbox"
+      value={filterValue}
+    />
+    <span className="pl-4">{filterName}</span>
```

Comment:
Nit: line break between siblings
```suggestion

    <span className="pl-4">{filterName}</span>
```